### ISSUE TITLE
validation/test error on CNNDM

### ISSUE THREAD

**Issue by monologue1107** [nlpyang/PreSumm#198]
After I trained sum model for CNNDM, I tried to run the validation on my ckpt.
The script I use are followed:
`export BERT_DATA_PATH=../bert_data/
export MODEL_PATH=../model/
python train.py \
  -task abs \
  -mode validate \
  -batch_size 3000 \
  -test_batch_size 500 \
  -bert_data_path ${BERT_DATA_PATH} \
  -log_file ../logs/val_abs_bert_cnndm \
  -model_path ${MODEL_PATH} \
  -sep_optim true \
  -use_interval true \
  -visible_gpus 1 \
  -max_pos 512 \
  -max_length 200 \
  -alpha 0.95 \
  -min_length 50 \
  -result_path ../logs/abs_bert_cnndm`

When running validation dataset, it went all well. While during test part, it didn't go further by:

`[2020-10-18 20:34:05,003 INFO] Loading valid dataset from ../bert_data/cnndm.valid.6.bert.pt, number of examples: 1362
[2020-10-18 20:34:35,629 INFO] Validation perplexity: 9.66616
[2020-10-18 20:34:35,630 INFO] Validation accuracy: 56.876
[2020-10-18 20:34:35,913 INFO] Loading checkpoint from ../model/model_step_158000.pt
[2020-10-18 20:34:39,414 INFO] loading configuration file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-config.json from cache at ../temp/4dad0251492946e18ac39290fcfe91b89d370fee250efe9521476438fe8ca185.7156163d5fdc189c3016baca0775ffce230789d7fa2a42ef516483e4ca884517
[2020-10-18 20:34:39,415 INFO] Model config {
  "architectures": [
    "BertForMaskedLM"
  ],
  "attention_probs_dropout_prob": 0.1,
  "finetuning_task": null,
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 512,
  "model_type": "bert",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "num_labels": 2,
  "output_attentions": false,
  "output_hidden_states": false,
  "pad_token_id": 0,
  "pruned_heads": {},
  "torchscript": false,
  "type_vocab_size": 2,
  "vocab_size": 30522
}

[2020-10-18 20:34:40,460 INFO] loading weights file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-pytorch_model.bin from cache at ../temp/aa1ef1aede4482d0dbcd4d52baad8ae300e60902e88fcb0bebdec09afd232066.36ca03ab34a1a5d5fa7bc3d03d55c4fa650fed07220e2eeebc06ce58d0e9a157
[2020-10-18 20:34:48,701 INFO] Loading test dataset from ../bert_data/cnndm.test.0.bert.pt, number of examples: 2001
[2020-10-18 20:34:49,884 INFO] loading file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-vocab.txt from cache at ../temp/26bc1ad6c0ac742e9b52263248f6d0f00068293b33709fae12320c0e35ccfbbb.542ce4285a40d23a559526243235df47c5f75c197f04f37d1a0c124c32c9a084`

Do you have some opinions about this kind of issue?

**Comment by beanandrew** [comment]
Yes, I was in the same condition when I try to test my model. Have you solved this problem?

**Comment by zyw12358** [comment]
> Yes, I was in the same condition when I try to test my model. Have you solved this problem?

Have you solved this problem?

**Comment by zyw12358** [comment]
> Yes, I was in the same condition when I try to test my model. Have you solved this problem?



> Yes, I was in the same condition when I try to test my model. Have you solved this problem?

你好，请问之后你解决这个问题了吗

**Comment by Camellia1110** [comment]
Have you solved it?

**Comment by Dengguty** [comment]
> Yes, I was in the same condition when I try to test my model. Have you solved this problem?
Hello, have you solved this problem since then?
你好，请问之后你解决这个问题了吗？

**Comment by Dengguty** [comment]
> Have you solved it?
Hello, have you solved this problem since then?
你好，请问之后你解决这个问题了吗？

**Comment by Dengguty** [comment]
> > Yes, I was in the same condition when I try to test my model. Have you solved this problem?
> 
> Have you solved this problem?

Hello, have you solved this problem since then?
你好，请问之后你解决这个问题了吗？

**Comment by Dengguty** [comment]
> Yes, I was in the same condition when I try to test my model. Have you solved this problem?

Hello, have you solved this problem since then?
你好，请问之后你解决这个问题了吗？

**Comment by WahyuBagus8545** [comment]
Have you all solve the problem?
